src/cloudwatch.ts

```diff
@@ -0,0 +1,148 @@
+import { ClientConfiguration as cwConfig, PutMetricDataInput } from "aws-sdk/clients/cloudwatch";
+import { ClientConfiguration as logConfig, FilterLogEventsRequest } from "aws-sdk/clients/cloudwatchlogs";
+import { CloudWatch, CloudWatchLogs } from "aws-sdk";
+import { DateTime } from "luxon";
+import { cwLogger } from "./handler";
+// @ts-ignore
+// tslint:disable-next-line:no-var-requires
+const AWSXRay = require("aws-xray-sdk");
+
+export class CW {
```

Comment:
The naming convention is not followed

Reply:
Naming the class CloudWatch would override the CloudWatch import above